toolkit/scripts/pkggen.mk

```diff
@@ -174,6 +175,7 @@ $(STATUS_FLAGS_DIR)/build-rpms.flag: $(preprocessed_file) $(chroot_worker) $(go-
 	$(go-scheduler) \
 		--input="$(preprocessed_file)" \
 		--output="$(built_file)" \
+		--output-build-state-csv-file="$(output_csv_file)" \
```

Comment:
just my 2 cents here, but I think we should probably just always output the file.  This argument could simply be an option to change the filename.